src/sdk/namespace_nb.js

```diff
@@ -312,6 +313,30 @@ class NamespaceNB {
     put_object_retention(params, object_sdk) {
         return object_sdk.rpc_client.object.put_object_retention(params);
     }
+
+    ////////////
+    //  ACLs  //
+    ////////////
+
+    async get_object_acl(params, object_sdk) {
+        await object_sdk.read_object_md({
+            bucket: params.bucket,
+            key: params.key,
+            version_id: params.versionId
+        });
+
+        return {
+            owner: s3_utils.DEFAULT_S3_USER,
+            access_control_list: [{
+                Grantee: { ID: s3_utils.DEFAULT_S3_USER, Type: "CanonicalUser" },
+                Permission: "FULL_CONTROL"
+            }]
+        };
+    }
+
+    put_object_acl(params, object_sdk) {
+        return object_sdk.rpc_client.object.put_object_acl(params);
```

Comment:
Since this api does not exist in object_api we want this flow to do what s3_put_object_acl used to do which is to read_object_md to detect if bucket/object is missing or other errors, and then ignore the acl.